File: BeFreeAPI/BeFreeAPI/Models/SubCategoria.cs
namespace BeFreeAPI.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("tbSubCategoria")]
    public partial class SubCategoria
    {
        [Key]
        public int idSubCategoria { get; set; }

        public int idCategoria { get; set; }

        public string descricao { get; set; }
    }
}

File: BeFree/app/src/main/java/tcc/befree/models/SubCategoria.java
package tcc.befree.models;

/**
 * Created by gabro on 20/05/2017.
 */

public class SubCategoria {
    public int idSubCategoria;
    public int idCategoria;
    public String descricao;

    @Override
    public String toString(){
        return this.descricao;
    }
}
